Dockerfile

```diff
@@ -11,6 +11,7 @@ WORKDIR /app
 
 COPY --chown=app:app package.json package-lock.json /app/
 COPY --chown=app:app assets/ /app/assets/
+RUN mkdir "logs"
```

Comment:
As discussed, I think this potentially could be removed and an empty directory used in the repo but fine.

Reply:
Interesting approach, and this of course works. In `lev-api` I distinctly recall 2 deviations here
1. `logs/` are not listed in the `.gitignore` - I don't think this matters, and should perhaps also be included into the `lev-api` repo for consistency
2. The application itself is generating this `logs/` folder as it's technically required for the file stream - perhaps we could implement something here similar?

Reply:
1. for logs/ I just checked it is in .gitignore for lev-api as well
2. in lev-web-ui, we are only using config to pass to hmpo-app at the very starting point of the app. hmpo-logger unfortunately does not create logs/ folder if it doesnt already exist, it will throw an error 'the file or folder doesn't exist' or otherwise we can have the log file on app level rather than in a separate logs/ folder and hmpo-logger would be able to create it.